software/cryptotvgen/README.md

```diff
@@ -53,8 +53,10 @@ See help for details and available options.
 ```
 $ cryptotvgen --prepare_libs --candidates_dir=software/dummy_lwc_ref
 ```
-Replace `software/dummy_lwc_ref` with the relative/absolute path to the subfolder containing `crypto_aead` and `crypto_hash` folders containing the reference implementation of an algorithm. 
-This could also be the root to an already extracted SUPERCOP distribution. The built libraries will be in then available in `software/dummy_lwc_ref/lib` folder.
+Replace `software/dummy_lwc_ref` with the relative path to the subfolder containing
```

Comment:
I just realized the old text has lots of typos and mistakes, but is the proposed change just the omission of "absolute" for the types of path? The path can also be an absolute path and not just relative, so that part I believe is the one thing right there :)

Reply:
You are correct. I do not know have the absolute path choked for me the first time I ran this. Just adds a little documentation about where the lib dir gets created.